core/src/main/scala/org/apache/spark/SparkConf.scala

```diff
@@ -203,6 +203,28 @@ class SparkConf(loadDefaults: Boolean) extends Cloneable with Logging {
     getOption(key).map(_.toBoolean).getOrElse(defaultValue)
   }
 
+  // Limit of bytes for total size of results (default is 1GB)
```

Comment:
I'd like another set of eyes on the change from @pwendell or @JoshRosen  . The reason I hesitate before merging is the large number of small changes and merge conflict potential. Although I do definitely think the variable names are improved by the suffix.

For this particular issue, maybe expose just `getMemory` from this class, and inline the two simple calls to it that currently use `getMaxResultSize`? writing `getMemory("spark.driver.maxResultSize", "1g", outputScale = 'b')` in two places isn't bad versus constructing a new home for it.